data/856/824/91/85682491.geojson

```diff
@@ -566,6 +566,8 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
+        "digitalenvoy:metro_code":276003,
```

Comment:
why is this gettting a metro_code and a region_code DE concordance? I think it should just get a region_code (since it's a region)?

Reply:
DigitalEnvoy "metro areas" are essentially grouped regions.. so each region code will map to a unique region record, and each metro code will map to one or more region.

Reply:
That seems okay for now... but we'd need to strip them out later once we have marketarea placetype coverage in Germany?